orchagent/vxlanorch.cpp

```diff
@@ -295,10 +295,17 @@ create_tunnel(
 void
 remove_tunnel(sai_object_id_t tunnel_id)
 {
-    sai_status_t status = sai_tunnel_api->remove_tunnel(tunnel_id);
-    if (status != SAI_STATUS_SUCCESS)
+    if(tunnel_id != SAI_NULL_OBJECT_ID)
+    {
+        sai_status_t status = sai_tunnel_api->remove_tunnel(tunnel_id);
+        if (status != SAI_STATUS_SUCCESS)
+        {
```

Comment:
Could you add an error message here?

Reply:
> Could you add an error message here?
The "runtime_error"  in the next line has thrown the error message.